ember-cli/2017-06/june-22.md

```diff
@@ -0,0 +1,9 @@
+# ember-cli Meeting Notes - 2017-06-22
+
+## Attendees
+
+
+## Agenda
```

Comment:
Would like to talk about:

1. https://github.com/ember-cli/rfcs/pull/108
2. Setting a date for FastBoot 1.0.